src/main/java/common/Constants.java

```diff
@@ -0,0 +1,13 @@
+package common;
+
+import java.util.regex.Pattern;
+
+public class Constants {
+	public static final int PER_PRICE = 1000;
```

Comment:
가독성을 위해 습관적으로 단위 언더바(_) 를 사용해봐도 좋을 것 같습니다 ~
```suggestion
	public static final int PER_PRICE = 1_000;
```